protostar/commands/declare/declare_command.py

```diff
@@ -12,7 +16,7 @@
 )
 
 
-class DeclareCommand(Command):
+class DeclareCommand(Command, SignableCommandMixin, NetworkCommandMixin):
```

Comment:
Let me quote you from hour ago:
`composition > inheritance`

Reply:
it's a mixin. it's not inheritance, inheritance is only a mechanism here to add functionality.

Reply:
i can mix the functionality in with some other mechanism - no problem, if you consider that an issue.

Reply:
https://en.wikipedia.org/wiki/Mixin

Reply:
As long as mixin doesn't have deps it should be ok.